src/util/common-util.ts

```diff
@@ -11,7 +11,12 @@ import moment from 'moment'
 export default class CommonUtils {
   // Formatting date in the desired format for displaying in the template
   static formatDisplayDate (date: Date, format?: string) {
-    return date ? moment(date).format(format || 'MMM DD, YYYY') : ''
+    if (date) {
+      const moment = require('moment-timezone')
```

Comment:
import 'moment-timezone'; at the top should suffice I believe

Reply:
yup looks like importing works, just wasn't sure that was an option cause the moment tz docs didn't give examples of imports.
https://momentjs.com/timezone/docs/

Reply:
Typescript vs Javascript pretty much